### ISSUE TITLE
jx-convert-jenkinsfile fail on the preview env creation

### ISSUE THREAD

**Issue by vbehar** [jenkins-x/jx#6521]
### Summary 

[jx-convert-jenkinsfile](https://github.com/jenkins-x/jx-convert-jenkinsfile) fails on a `Jenkinsfile` with a cmd to create a preview env.

### Steps to reproduce the behavior

Run version 0.0.1 on the following `Jenkinsfile`:

```
pipeline {
    stages {
        stage('stage') {
            steps {
                dir ('./charts/preview') {
                    container('jx') {
                        sh 'make'
                        sh 'jx preview --namespace "${PREVIEW_NAME}" --name "${PREVIEW_NAME}" --release "${PREVIEW_NAME}" --app ${APP_NAME} --dir ../.. --verbose'
                    }
                }
            }
        }
    }
}
```

### Expected behavior

no error

### Actual behavior

```
Error parsing Jenkinsfile:  Jenkinsfile Jenkinsfile cannot be parsed. It may contain code outside of the pipeline {} block, or it may not have a pipeline {} block at all.: 8:28: invalid syntax: "\"jx preview --namespace \"${PREVIEW_NAME}\" --name \"${PREVIEW_NAME}\" --release \"${PREVIEW_NAME}\" --app ${APP_NAME} --dir ../.. --verbose\""
```

**Comment by abayer** [comment]
Aaaaah - I'm pretty sure this is a string quoting issue.

**Comment by abayer** [comment]
Fixed in v0.0.2, I hope. =)
